src/main/java/org/isf/telemetry/daemon/TelemetryDaemon.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Open Hospital (www.open-hospital.org)
+ * Copyright © 2006-2022 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
```

Comment:
Shouldn't all new files have 2023 as the copyright date?
```suggestion
 * Copyright © 2006-2023 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
```